spec/features/idv/analytics_spec.rb

```diff
@@ -0,0 +1,97 @@
+      'IdV: doc auth document_capture submitted' => { success: false, errors: { front_image: ['Please fill in this field.'], back_image: ['Please fill in this field.'] }, is_fallback_link: false, error_details: { front_image: [:blank], back_image: [:blank] }, flow_path: 'standard', step: 'document_capture', step_count: 1 },
+      'IdV: doc auth ssn visited' => { flow_path: 'standard', step: 'ssn', step_count: 1 },
+      'IdV: doc auth ssn submitted' => { success: true, errors: {}, flow_path: 'standard', step: 'ssn', step_count: 1 },
+      'IdV: doc auth verify visited' => { flow_path: 'standard', step: 'verify', step_count: 1 },
+      'IdV: doc auth verify submitted' => { success: true, errors: {}, flow_path: 'standard', step: 'verify', step_count: 1 },
+      'IdV: doc auth verify_wait visited' => { flow_path: 'standard', step: 'verify_wait', step_count: 1 },
+      'IdV: doc auth optional verify_wait submitted' => { success: true, errors: {}, address_edited: false, proofing_results: { messages: [], exception: nil, transaction_id: 'resolution-mock-transaction-id-123', reference: 'aaa-bbb-ccc', timed_out: false, context: { dob_year_only: false, should_proof_state_id: true, stages: { resolution: { client: 'ResolutionMock', errors: {}, exception: nil, success: true, timed_out: false, transaction_id: 'resolution-mock-transaction-id-123', reference: 'aaa-bbb-ccc' }, state_id: { client: 'StateIdMock', errors: {}, success: true, timed_out: false, exception: nil, transaction_id: 'state-id-mock-transaction-id-456', state: 'MT', state_id_jurisdiction: 'ND' } } } }, ssn_is_unique: true, step: 'verify_wait_step_show' },
+      'IdV: phone of record visited' => {},
+      'IdV: phone confirmation form' => { success: true, errors: {}, phone_type: :mobile, types: [:fixed_or_mobile], carrier: 'Test Mobile Carrier', country_code: 'US', area_code: '202' },
+      'IdV: phone confirmation vendor' => { success: true, errors: {}, vendor: { messages: [], exception: nil, context: { stages: [{ address: 'AddressMock' }] }, transaction_id: 'address-mock-transaction-id-123', timed_out: false }, new_phone_added: false },
+      'IdV: review info visited' => {},
+      'IdV: review complete' => {},
+      'IdV: final resolution' => { success: true },
+      'IdV: personal key visited' => {},
+      'IdV: personal key submitted' => {},
+    }
+    {
+      FSMv1: common_events.merge(
+        'Frontend: IdV: show personal key modal' => {},
+      ),
+      FSMv2: common_events.merge(
+        'IdV: personal key confirm visited' => {},
+        'IdV: personal key confirm submitted' => {},
+      ),
+    }
+  end
+  # rubocop:enable Layout/LineLength
+
+  before do
+    allow_any_instance_of(ApplicationController).to receive(:analytics).and_return(fake_analytics)
+  end
+
+  {
+    FSMv1: [],
+    FSMv2: %w[personal_key personal_key_confirm],
```

Comment:
Should we include `password_confirm`?

Reply:
Per my previous comment, I only did this PR with personal key since it was the only step fully merged when I made this.

Reply:
included password_confirm in cc24137